controllers/customer_controller_test.go

```diff
@@ -24,6 +25,9 @@ var _ = Describe("Customer Controller", func() {
 	Describe("Create new Jira Service Desk customer", func() {
 		Context("With valid fields", func() {
 			It("should create a new customer", func() {
+				// Randomize customer name and email
+				mockData.SampleCustomer = cUtil.RandCustomer(mockData.SampleCustomer)
```

Comment:
This random customer should be created outside the describe block. In current case, the after each block won't be able to delete it after this test case has run